integration-tests/cucumber-tests-execution/src/main/java/org/opensmartgridplatform/cucumber/execution/AppBase.java

```diff
@@ -40,14 +44,19 @@ public static int run(final AppBase app, final String[] testClasses, final Strin
             p.parseArgument(args);
             LOGGER.info("Start");
             final int retval = app.runTests(testClasses);
-            LOGGER.info("End. Retval = {}", retval);
+            LOGGER.info(END_RETVAL, retval);
 
             return retval;
         } catch (final CmdLineException e) {
             LOGGER.error("Incorrect usage", e);
             LOGGER.error("java -jar <...>.jar [opts] ...");
             p.printUsage(System.err);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS106)

Reply:
The library has no other functions and uses System.out and System.err.